casepro/utils/__init__.py

```diff
@@ -59,6 +60,23 @@ def json_encode(data):
     return json.dumps(data, cls=JSONEncoder)
 
 
+def json_decode(data):
+    """
+    Decodes the given JSON as primitives
+    """
+    if isinstance(data, six.binary_type):
+        data = data.decode('utf-8')
```

Comment:
@rudigiesler you're right about `json.dumps` - I just added some tests to verify that https://github.com/rapidpro/casepro/blob/50977404346206ccbe93a8647761bf5f63d583d4/casepro/utils/tests.py#L92-L92

Do you guys want to try deploying this branch to a test server before it is merged? I was going to likewise to check RapidPro integration.